src/yuzu/main.cpp

```diff
@@ -1035,19 +1035,36 @@ void GMainWindow::BootGame(const QString& filename) {
     }
 
     const u64 title_id = Core::System::GetInstance().CurrentProcess()->GetTitleID();
+    const FileSys::PatchManager patch_manager(title_id);
+    QString dlc;
+
+    for (const auto& kv : patch_manager.GetPatchVersionNames(nullptr)) {
+        if (kv.first == "DLC" && !kv.second.empty()) {
+            dlc = QString::fromStdString("DLC " + kv.second);
+        }
+    }
+
+    if (dlc.isEmpty()) {
+        dlc = QString::fromStdString("No DLC");
+    }
 
     std::string title_name;
+    QString title_version;
     const auto res = Core::System::GetInstance().GetGameName(title_name);
-    if (res != Loader::ResultStatus::Success) {
-        const auto metadata = FileSys::PatchManager(title_id).GetControlMetadata();
-        if (metadata.first != nullptr)
-            title_name = metadata.first->GetApplicationName();
+    const auto metadata = FileSys::PatchManager(title_id).GetControlMetadata();
+    if (metadata.first != nullptr) {
+        title_version = QString::fromStdString(metadata.first->GetVersionString());
 
-        if (title_name.empty())
-            title_name = FileUtil::GetFilename(filename.toStdString());
+        if (res != Loader::ResultStatus::Success) {
+            title_name = metadata.first->GetApplicationName();
+            if (title_name.empty()) {
+                title_name = FileUtil::GetFilename(filename.toStdString());
+            }
+        }
     }
     LOG_INFO(Frontend, "Booting game: {:016X} | {}", title_id, title_name);
-    UpdateWindowTitle(QString::fromStdString(title_name));
+    UpdateWindowTitle(QString::fromStdString(title_name),
+                      QString::fromStdString(std::to_string(title_id)), title_version, dlc);
```

Comment:
Represent title ID in hexadecimal

```suggestion
                      QString::fromStdString(fmt::format("{:016X}", title_id), title_version, dlc);
```

Reply:
Converted it in UpdateWindowTitle